ansible_collections/arista/avd/roles/eos_designs_to_containerlab/tasks/deploy.yml

```diff
@@ -0,0 +1,237 @@
+---
+#tasks file for containerlab hosts
+
+- ansible.builtin.setup:
+    # gather_subset:
+    #   - '!all'
+    filter:
+      - 'ansible_env'
+
+- name: Check if more than one containerlab host is defined
+  set_fact:
+    multinode: "{{ true if groups['CONTAINERLAB']|length > 1 else false }}"
```

Comment:
this will error out if the group `CONTAINERLAB` does not exist. Maybe use `true if groups['CONTAINERLAB'] | arista.avd.default([]) | length > 1 else false`

Reply:
changed as suggested